From: Aurelien Jarno <aurel32@debian.org>
    
mesa fails to build on GNU/kFreeBSD, since some parts are not enabled.
    
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=524690

Index: mesa/configure.ac
===================================================================
--- mesa.orig/configure.ac	2009-09-29 11:44:55.000000000 +0200
+++ mesa/configure.ac	2009-09-29 11:45:12.000000000 +0200
@@ -736,7 +736,7 @@
             ;;
         esac
         ;;
-    freebsd* | dragonfly*)
+    freebsd* | dragonfly* | kfreebsd*-gnu*)
         DEFINES="$DEFINES -DPTHREADS -DUSE_EXTERNAL_DXTN_LIB=1"
         DEFINES="$DEFINES -DIN_DRI_DRIVER -DHAVE_ALIAS"
         DEFINES="$DEFINES -DGLX_INDIRECT_RENDERING"
